Basic app and server

KalleHallden opened the pull request:
This pull request contains the initial commit of a mobile application for requesting actions from the car which is needed as to "[Create Basic Interface app for steering](https://github.com/DIT112-V19/group-04/issues/3)"

Commit:
First commit

Commit:
First commit

Commit:
Created basic interface app for steering

platisd reviewed (changes requested):
What story does this pull request solve or is related to? Please specify in the pull request description so we can achieve full traceability.

hultena committed:
Start of the vehicle logic

I started by creating some classes.
Not sure if this is acceptable python code or not.

hultena committed:
Additional work on the vehicle logic

Added some functionality for the vehicle to follow vectors.

konradotto commented:
Edited the description adding a link to the issue (don't see a way of linking issues directly, only milestones)

hultena committed:
Fixed some issues

platisd commented:
> Edited the description adding a link to the issue (don't see a way of linking issues directly, only milestones)

@konradotto do you mean you cannot write that it "relates to #3"? :grin:
